File: ads3/poo/Semana1/aula05/c++/Program.cpp
#include <iostream>
using namespace std;

int main(int argc, char *argv[]){
	int qtdePedido = 10;
	int qtdeEstoque = 20;
	int qtdeMinimaPorVenda = 5;
	
	cout << "Entre com o valor do pedido: ";
	cin >> qtdePedido;
		
	cout << "Entre com o valor do estoque: ";
	cin >> qtdeEstoque;
		
	cout << "Entre com a quantidade minima por venda: ";
	cin >> qtdeMinimaPorVenda;

	bool vendaLiberada = qtdePedido <= qtdeEstoque && qtdePedido >= qtdeMinimaPorVenda;
	cout << "Venda Liberado: " << (vendaLiberada ? "true" : "false") << endl;
		
	return 0;
}

File: ads3/poo/Semana1/aula05/java/Program.java
import java.util.Scanner;
public class Program{
	public static void main(String[] args){
		Scanner entrada = new Scanner(System.in);
		int qtdePedido;
		int qtdeEstoque;
		int qtdeMinimaPorVenda;
		
		System.out.print("Entre com o valor do pedido: ");
		qtdePedido = entrada.nextInt();
		
		System.out.print("Entre com o valor do estoque: ");
		qtdeEstoque = entrada.nextInt();
		
		System.out.print("Entre com a quantidade minima por venda: ");
		qtdeMinimaPorVenda = entrada.nextInt();
		
		boolean vendaLiberada = qtdePedido <= qtdeEstoque && qtdePedido >= qtdeMinimaPorVenda;
		System.out.println("Venda Liberado: "+vendaLiberada);
		

	}
}
